### ISSUE TITLE
Update to MAMP 3.4?

### ISSUE THREAD

**Issue by shaneparsons** [majksner/php-phalcon-mamp#20]
I can't get Phalcon working on MAMP 3.4 (PHP 5.6.10) and I'm wondering if that because it's not yet meant for 5.4?

Thanks

**Comment by nurtext** [comment]
The MAMP version doesn't matter at all, but PHP version is important. You can't use Phalcon builds for PHP 5.4 on MAMP with PHP 5.6.10. Make sure you choose the right file that suits your MAMP environment.

However please provide more information about what is not working. I've just submitted a new PR for Phalcon 2.0.7 and PHP 5.6.10

**Comment by shaneparsons** [comment]
Hey, sorry for the delay.

I can't exactly remember what the problem was, but I'm pretty sure I couldn't get past the **checking your installation** steps on https://docs.phalconphp.com/en/latest/reference/tutorial.html#checking-your-installation.

I realized it was a problem with the mamp setup when I brought the exact same code into OSX's native localhost and everything was working fine.

It looks like the build I was using has been updated via your PR, so I'll give it a shot when I get a chance and keep you posted.
